utils/download_model.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import wget
+
+def get_model_path():
```

Comment:
Maybe we'd like to download other models in future. Can you refactor the URL out of this function so that is accepts that as input?